### PR TITLE
Feature/likernft

### PR DESCRIPTION
**Pull request by williamchong**

### CODE REVIEW

File: src/routes/likernft/history.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        { data: sendData },
+      ] = await Promise.all([
+        axios.get(`${COSMOS_LCD_INDEXER_ENDPOINT}/cosmos/tx/v1beta1/txs?events=likechain.likenft.EventNewClass.class_id=%27%22${classId}%22%27`),
+        axios.get(`${COSMOS_LCD_INDEXER_ENDPOINT}/cosmos/tx/v1beta1/txs?events=likechain.likenft.EventMintNFT.class_id=%27%22${classId}%22%27`),
+        axios.get(`${COSMOS_LCD_INDEXER_ENDPOINT}/cosmos/tx/v1beta1/txs?events=cosmos.nft.v1beta1.EventSend.class_id=%27%22${classId}%22%27`),
+      ]);
+      const list = [];
+      list.concat(newClassData.tx_responses || []);
+      list.concat(mintData.tx_responses || []);
+      list.concat(sendData.tx_responses || []);
```
Comment (nnkken) [inline src/routes/likernft/history.js]: Any deduplication needed?

**Review by nnkken — APPROVED** [review]

File: src/routes/likernft/metadata.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+  '/metadata/image/class_(:classId)(.png)?',
+  async (req, res, next) => {
+    try {
+      const { classId } = req.params;
+      const doc = await getISCNDocByClassId(classId);
+      const classDocRef = await doc.ref.collection('class').doc(classId).get();
+      const classData = classDocRef.data();
+      // const iscnRef = queryRef.parent.parent;
+      // const iscnDocRef = iscnDataRef.get();
+      // const iscnData = await iscnDocRef();
```
Comment (WeiJun0827) [inline src/routes/likernft/metadata.js]: Should we keep code here?

**Review by WeiJun0827 — APPROVED** [review]

File: src/routes/likernft/metadata.js (hunk 2 above)
Comment (williamchong) [reply src/routes/likernft/metadata.js]: Just an example for later implementation

File: src/routes/likernft/history.js (hunk 1 above)
Comment (williamchong) [reply src/routes/likernft/history.js]: yes and in fact wrong syntax

File: src/routes/likernft/metadata.js (hunk 2 above)
Comment (WeiJun0827) [reply src/routes/likernft/metadata.js]: I see

**Pull request merged**